google/resource_compute_shared_vpc_host.go

```diff
@@ -0,0 +1,91 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceComputeSharedVpcHost() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeSharedVpcHostCreate,
+		Read:   resourceComputeSharedVpcHostRead,
+		Delete: resourceComputeSharedVpcHostDelete,
+
+		Schema: map[string]*schema.Schema{
+			"project": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceComputeSharedVpcHostCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	projectID, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	op, err := config.clientCompute.Projects.EnableXpnHost(projectID).Do()
+	if err != nil {
+		return fmt.Errorf("Error disabling XPN Host: %s", err)
```

Comment:
Typo: We are enabling here, not disabling. 

Consider using "Shared VPC" instead of XPN since this is the nomenclature we are using in our docs.

Maybe: "Error enabling shared vpc host for project %s: %s"

Reply:
Done